src/app/src/main/kotlin/org/vaccineimpact/api/app/logic/BurdenEstimateLogic.kt

```diff
@@ -284,6 +287,8 @@ class RepositoriesBurdenEstimateLogic(private val modellingGroupRepository: Mode
         {
             burdenEstimateRepository.updateBurdenEstimateSetFilename(setId, filename)
         }
+
+        notifier.notify("A burden estimate set has just been uploaded for ${group.id}")
```

Comment:
Would it be worth putting more information in the message about the burden estimate? Since Kim doesn't have visibility on Touchstone progress yet, maybe scenario would be a useful thing to surface here (would at least show if multiple uploads are multiple attempts at the same set, or different scenarios). Touchstone is probably limited usefulness since I think there's only usually one active one, but could possibly be of interest. 
Also, would it be worth showing if the upload was valid or invalid?

Reply:
Both great points